Flaky TestIngester_inflightPushRequestsBytes

```
--- FAIL: TestIngester_inflightPushRequestsBytes (6.54s)
    ingester_test.go:8791: 1 100000 69.868369ms
    ingester_test.go:8791: 1 2100000 1.422553747s
    ingester_test.go:8738: expected 1, got 0
    ingester_test.go:8766: 
                Error Trace:    /go/src/github.com/grafana/mimir/pkg/ingester/ingester_test.go:8766
                Error:          Received unexpected error:
                                 # HELP cortex_ingester_instance_rejected_requests_total Requests rejected for hitting per-instance limits
                                 # TYPE cortex_ingester_instance_rejected_requests_total counter
                                 cortex_ingester_instance_rejected_requests_total{reason="ingester_max_inflight_push_requests"} 0
                                -cortex_ingester_instance_rejected_requests_total{reason="ingester_max_inflight_push_requests_bytes"} 0
                                +cortex_ingester_instance_rejected_requests_total{reason="ingester_max_inflight_push_requests_bytes"} 3
                                 cortex_ingester_instance_rejected_requests_total{reason="ingester_max_inflight_read_requests"} 0
                                 cortex_ingester_instance_rejected_requests_total{reason="ingester_max_ingestion_rate"} 0
                                 cortex_ingester_instance_rejected_requests_total{reason="ingester_max_series"} 0
                                 cortex_ingester_instance_rejected_requests_total{reason="ingester_max_tenants"} 0
                                 
                Test:           TestIngester_inflightPushRequestsBytes
method=Ingester.Shipper.Sync level=info msg="uploading new block to long-term storage" block=00000000010000000000000000
method=Ingester.Shipper.Sync level=debug msg="uploaded file" from=/tmp/TestShipper_DeceivingUploadErrors1560796317/001/00000000010000000000000000/index dst=00000000010000000000000000/index bucket="fs: /tmp/TestShipper_DeceivingUploadErrors1560796317/002"
method=Ingester.Shipper.Sync level=error msg="uploading new block to long-term storage failed" block=00000000010000000000000000 err="failed to upload meta: base name matches, will fail upload"
```

--- a/pkg/ingester/ingester_test.go
+++ b/pkg/ingester/ingester_test.go
@@ -8852,7 +8852,7 @@ func TestIngester_inflightPushRequestsBytes(t *testing.T) {
 		}
 
 		require.Eventually(t, func() bool {
-			return i.inflightPushRequestsBytes.Load() > 0
+			return i.inflightPushRequestsBytes.Load() == int64(requestSize)
 		}, targetRequestDuration/3, 3*time.Millisecond)
 
 		require.NoError(t, testutil.GatherAndCompare(reg, strings.NewReader(fmt.Sprintf(`

